requirements.txt

```diff
@@ -0,0 +1 @@
+click
```

Comment:
Missing import for `rpi_ws281x`

Reply:
As i know, this lib you can not just install via pip so i don´t know if such an entry would lead to some error while executing `sudo python3 -m pip install -r requirements.txt`.

The plan is to copy / write custom manual how to setup the raspberry pi for WS281x and clone / fork the wanted libraries so this project still work even if other repositories are removed

Reply:
Ok, I understand. Could you please add the manual within this PR, as we're introducing the library here, aswell?

Reply:
Resolved, as we added the links to the original docs for now.